uaclient/event_logger.py

```diff
@@ -46,6 +49,9 @@ def __init__(self):
         self._processed_services = set()  # type: Set[str]
         self._failed_services = set()  # type: Set[str]
         self._needs_reboot = False
+        self._command = ""
+        self._output_format = "json"
```

Comment:
Rather than add a new `_output_format` field, I suggest changing the `EventLoggerMode` enum to have different values: i.e. `CLI`, `JSON`, and `YAML`

Reply:
After playing for it a bit, I would like to retreat from this change. The EventLoggerMode is responsible for determining a change in the behavior, while the format is just an output detail in my point of view. I understand how we have the `--format` option to the commands, but reflecting the same idea to the class led to some confusion imho.

Reply:
I see that we will have one less property in the class, but the logic to process events will be practically the same. I personally think it is more readable to have a mode (plain text on CLI x machine-readable output), and then defining the format based on what is in the CLI. But I can change this, no problem.

Reply:
For the simplest way, I ended up implementing the original request. I think it can be like this for now, and the overall logic can be improved when we make a decision on how the overall JSON responses should work.